### PR TITLE
Add yards to length conversions

### PR DESCRIPTION
**Pull request by logansparlin**

### CODE REVIEW

File: lib/definitions/length.js [hunk 1]
```diff
@@ -40,6 +40,13 @@ imperial = {
     }
   , to_anchor: 1/12
   }
+, yd: {
+    name: {
+      singular: 'Yard'
+    , plural: 'Yards'  
+    }
+  , to_anchor: 1/0.3333
```
Comment (ben-ng) [inline lib/definitions/length.js]: This seems like a mistake: the anchor is meters, so I was expecting `to_anchor: 0.9144`.

Comment (logansparlin) [inline lib/definitions/length.js]: Wouldn't the yards anchor be to feet? 1 foot = 3 yards so it seems like to_anchor should be equal to 3 here. That seems to product the correct results.

Comment (ben-ng) [inline lib/definitions/length.js]: You're right, I forgot that we had an anchor for each system of units.

**Comment by ben-ng** [comment]
Published in 1.3.0. Thank you!

**Pull request merged**